README.md

```diff
@@ -80,6 +80,7 @@ Through JNI extensions you can run NiFi processors using NARs. The JNI extension
 | SQLLite | [ExecuteSQL](PROCESSORS.md#executesql)<br/>[PutSQL](PROCESSORS.md#putsql)      |    -DENABLE_SQLITE=ON  |
 | Tensorflow | [TFApplyGraph](PROCESSORS.md#tfapplygraph)<br/>[TFConvertImageToTensor](PROCESSORS.md#tfconvertimagetotensor)<br/>[TFExtractTopLabels](PROCESSORS.md#tfextracttoplabels)<br/>      |    -DENABLE_TENSORFLOW=ON  |
 | USB Camera | [GetUSBCamera](PROCESSORS.md#getusbcamera)     |    -DENABLE_USB_CAMERA=ON  |
+| AWS | [ConsumeMQTT](PROCESSORS.md#consumeMQTT)<br/>[PublishMQTT](PROCESSORS.md#publishMQTT)     |    -DENABLE_AWS=ON  |
```

Comment:
s/ConsumeMQTT/AWS/ ?

Reply:
Updated by creating a new file for controller services. CONTROLLERS.md and placing the related AWSCredentialsService documentation in there and then linking the README.md to that.